modules/dists/SparseBlockDist.chpl

```diff
@@ -298,8 +298,9 @@ class LocSparseBlockDom {
   param stridable: bool;
   type sparseLayoutType;
   var parentDom: domain(rank, idxType, stridable);
-  var sparseDist = new unmanaged sparseLayoutType; //unresolved call workaround
-  var mySparseBlock: sparse subdomain(parentDom) dmapped new dmap(sparseDist);
+  var sparseDist = if _to_borrowed(sparseLayoutType) == DefaultDist then defaultDist
+                   else new dmap(new unmanaged sparseLayoutType); //unresolved call workaround
```

Comment:
> `//unresolved call workaround`

I know this comment was here before you touched this, but do you know what this is referring to?

Reply:
No idea :(

Reply:
I tracked this one down to my first PR. Will investigate further soon.